stories/common/Using_the_Kit.mdx

```diff
@@ -0,0 +1,76 @@
+import './css/docs.css';
+
+# Getting Started with the Toolkit
+
+## How to use the Components
+
+<!-- Due to how MDX v1 handles JSX & MD, any lines with props need to be in JSX -->
+<!-- see https://github.com/facebook/docusaurus/issues/6072#issuecomment-989516658 -->
+<!-- MDX v2 is optional as of Storybook v6.5, so there is potential for refactoring in the future -->
+
+<ol>
+  <li>Install the package: <code>npm i @button-inc/{props.theme}-theme</code></li>
+  <li>Import the components: <code>import Card from @button-inc/{props.theme}-theme/Card</code></li>
```

Comment:
1. Thank you for changing the example to `Card` instead of `Button`! I was perpetually confused by it since it's also the company name
2. Something's not quite working the props; in the Button theme I'm seeing installation instructions for the bcgov package: 
![image](https://user-images.githubusercontent.com/77306764/199835089-9a037938-2a2c-45c3-8ae9-07bcb661569a.png)